encoding/src/bytes.rs

```diff
@@ -10,7 +10,7 @@ use serde_bytes::ByteBuf;
 pub struct BytesSer<'a>(#[serde(with = "serde_bytes")] pub &'a [u8]);
 
 /// Wrapper for deserializing dynamic sized Bytes.
-#[derive(Deserialize, Serialize)]
+#[derive(Deserialize)]
```

Comment:
optional: to include Serialize on these deserialize types

It's nice to have it symmetric if you need to use one type for serializing and deserializing (no actual examples in code rn but can be useful to move values instead of serializing a reference to them).